_posts/2016-10-15-recurse-center-week-3.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: post
+title: "Week[2] @ The Recurse Center 🌻"
+comments: true
+categories: tech recurse
+---
+
+1/4 of the way through! Time is flying.
+This post is also quite late. I suppose it's a good sign that I've been too busy to write.
+
+# Overall
+
+The week started off as a low, and steadily improved to wonderful levels. 😊
```

Comment:
Yesh! A lot of this is straight from the brain. Was going to say that there's probably some weird issues with the sentences. So I appreciate these catches as it will alleviate my fears that those have persisted 😄